scripts/PokemonTower6F.asm

```diff
@@ -59,18 +60,18 @@ PokemonTower6Script4:
 	ld [wJoyIgnore], a
 	ld a, [wBattleResult]
 	and a
-	jr nz, .asm_60b82
+	jr nz, .did_not_defeat
 	SetEvent EVENT_BEAT_GHOST_MAROWAK
-	ld a, $7
+	ld a, TEXT_POKEMONTOWER6F_MAROWAK_DEPARTED
 	ldh [hSpriteIndexOrTextID], a
 	call DisplayTextID
 	xor a
 	ld [wJoyIgnore], a
-	ld a, $0
+	ld a, SCRIPT_POKEMONTOWER6F_DEFAULT ; useless
```

Comment:
Even though this is technically useless/redundant because of the `xor a`, I don't think this comment is worth adding; it looks too much like "setting the script to default is useless".